config/settings/base.py

```diff
@@ -296,6 +295,9 @@
     "thead": {
         "class": "table-secondary",
     },
+    "tfoot": {
+        "class": "table-light fw-bold",
```

Comment:
If it is possible to only bold the word "total" and not the numbers, that would be awesome, otherwise this is fine.